pages/wi/session/voting/start.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import Head from 'next/head';
+import styled from 'styled-components';
+import { Grid, Cell } from 'styled-css-grid';
+
+import { below } from '../../../../utilities';
+import ContentSection from '../../../../components/shared/ContentSection';
+
+import Header from '../../../../components/Session/Voting/Start/Header';
+
+const MainGrid = styled(Grid)`
+  grid-gap: 2.5rem;
+
+  ${below.med`
+    display: block;
+    grid-gap: 0;
+    margin-top: -5rem;
+  `};
+
+  h3 {
+    font-weight: 100;
+    margin-bottom: 1rem;
+  }
+
+  ul {
+    list-style: none;
+    padding-inline-start: 2rem;
+  }
+
+  ul li::before {
+    content: '\\2022';
+    color: red;
+    font-weight: bold;
+    display: inline-block;
+    width: 1em;
+    margin-left: -1em;
+  }
+`;
+
+const MainContent = styled(ContentSection)`
+  padding-top: 0;
+
+  ${below.med`
+    padding-top: 3rem;
+  `};
+`;
+
+const VotingStart = ({ user, loading }) => {
+  return (
+    <div>
+      <Head>
```

Comment:
We now have `NextSeo` in play for the title and description. All you need to do is import in and use instead of `Head` here. You can then add a unique description to it so we can get better SEO juices.